include/Util/helper.php

```diff
@@ -129,7 +129,7 @@ function join_array_elements_with_delimiter( $value, $delimiter = ',' ) {
 		return implode( $delimiter, $value );
 	}
 
-	return $value;
+	return is_string( $value ) ? $value : '';
```

Comment:
Note from Scrutinizer

`The condition is_string($value) is always true.`

Reply:
The same note applies here too.